IdentityCore/src/network/response_serializer/MSIDResponseSerialization.h

```diff
@@ -25,15 +25,18 @@
 
 @protocol MSIDRequestContext;
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
nit: do we need this? seems like eveything is nullable, so perhaps simply remove this with nullability identifiers?

Reply:
removed it.